src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallLibertyTask.groovy

```diff
@@ -134,4 +135,38 @@ class InstallLibertyTask extends AbstractTask {
             }
         }
     }
+    
+    
+    protected boolean isInstallationUpToDate() {
```

Comment:
consider renaming this method. you can probably pull off at least isInstallUpToDate or installUpToDate. maybe even installUpdateCheck